tools/docker-compose/receptor.conf

```diff
@@ -0,0 +1,27 @@
+---
+- log-level: debug
+
+- control-service:
+    service: control
+    filename: /var/run/receptor/receptor.sock
+
+- local-only:
+
+- work-command:
+    worktype: worker
+    command: ansible-runner
+    params: worker
+
+# - tls-client:
+#     name: insecure
+#     insecureskipverify: true
+
+# - ws-peer:
+#     tls: insecure
+#     address: wss://receptor-receptor.apps.tower.ocp4.testing.ansible.com
+
+- work-kubernetes:
+    worktype: ocp
+    namespace: receptor
+    image: quay.io/shanemcd/ee
```

Comment:
Ultimately this will need to come from the execution environment model?

That question might be getting too far ahead of things, and if it is, that's fine.